webpack.config.js

```diff
@@ -106,7 +106,7 @@ module.exports = function (env) {
 
     // bundle the client for webpack-dev-server
     // and connect to the provided endpoint
-    'webpack-dev-server/client?http://' + host + ':' + port,
+    `webpack-dev-server/client?http://${host}:${port}`,
```

Comment:
the only reason I concatenated is because people might not be using node 6+ to run webpack